src/index.js

```diff
@@ -1,31 +1,31 @@
 import _, { divide } from 'lodash'; // eslint-disable-line no-unused-vars
+import * as statusModule from './status';
 import './style.css';
 
-const toDoTasks = [
-  { desc: 'Reading a book', completed: true, index: 1 },
-  { desc: 'completing task', completed: true, index: 2 },
-  { desc: 'Climbing mountain', completed: true, index: 3 },
-];
-
 const tasks = document.querySelector('.list');
 
-function createTask(e) {
+// creates a view for task t
```

Comment:
- Kindly fix this error displayed in console:
<img width="776" alt="Screen Shot 2021-09-29 at 18 19 59" src="https://user-images.githubusercontent.com/22258274/135361477-329914f6-3d9f-40b8-a070-6fc11d96fde9.png">

Reply:
On my computer, everything works perfectly. Can you tell me when those errors appear?

Reply:
![Screenshot from 2021-09-30 12-09-09](https://user-images.githubusercontent.com/51750930/135409021-9e13e88a-59c2-408d-a6ce-b3483c729e05.png)